build-common/NHibernate.props

```diff
@@ -13,18 +13,26 @@
     <FileVersion Condition="'$(VersionSuffix)' != '' AND '$(BuildNumber)' != ''">$(VersionPrefix).$(BuildNumber)</FileVersion>
     <FileVersion Condition="'$(FileVersion)' == ''">$(VersionPrefix).0</FileVersion>
 
-    <NhAppTargetFrameworks Condition ="$(NhAppTargetFrameworks) == ''">net461;netcoreapp2.0</NhAppTargetFrameworks>
+    <NhAppTargetFrameworks Condition ="$(NhAppTargetFrameworks) == ''">net461;net6.0</NhAppTargetFrameworks>
     <NhLibTargetFrameworks Condition ="$(NhLibTargetFrameworks) == ''">net461;netcoreapp2.0;netstandard2.0</NhLibTargetFrameworks>
     <NhNetFx>false</NhNetFx>
     <NhNetFx Condition="$(TargetFramework.StartsWith('net4'))">true</NhNetFx>
-    <DefineConstants Condition="$(NhNetFx) AND $(NhVbNet) == ''" >NETFX;$(DefineConstants)</DefineConstants>
+    <!-- Visual Basic requires to use coma (,) as a separator, other project types - semicolon (;) -->
+    <DefineConstants Condition="$(NhNetFx) AND '$(MSBuildProjectExtension)' != '.vbproj'">NETFX;$(DefineConstants)</DefineConstants>
+    <DefineConstants Condition="$(NhNetFx) AND '$(MSBuildProjectExtension)' == '.vbproj'">NETFX,$(DefineConstants)</DefineConstants>
 
     <Product>NHibernate</Product>
     <Company>NHibernate.info</Company>
     <Copyright>Licensed under LGPL.</Copyright>
     <Authors>NHibernate community, Hibernate community</Authors>
     <NeutralLanguage>en-US</NeutralLanguage>
 
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <TreatSpecificWarningsAsErrors />
+    
+    <DisableImplicitPackageTargetFallback>True</DisableImplicitPackageTargetFallback>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(IsPackable)' == 'True'">
```

Comment:
What  this `$(IsPackable)` about? I don't think it works properly. At least it seems all this information is now missing in generated nuget package

Reply:
:-(

Reply:
@bahusoid. `'$(IsPackable)'` evaluates to `'true'` instead of `'True'`.

Reply:
Should be fixed here: https://github.com/nhibernate/nhibernate-core/pull/3125